admin_manual/configuration_user/user_configuration.rst

```diff
@@ -193,7 +193,8 @@ Hover your cursor over their name on the **Users** page until the "..."-menu ico
 appears at the far right. After clicking on it, you will see the **Disable** option.
 
 The user will not longer be able to access their Nextcloud until you enable them again.
-Keep in mind that the files, which were shared by this user will not longer be accessible.
+Keep in mind that the files, which were publicly shared by link by this user will no longer be accessible.
```

Comment:
what about email?

Reply:
No idea :shrug: 
Email sharing is just link sharing with an email sending added no?

Reply:
Yes but maybe make it more explicit

Reply:
"keep in mind" sounds like something unexpected that user/admin have to remember.
But actually that is the expected behaviour.
So maybe phrase it like this:
> The user will not longer be able to access their Nextcloud until you enable them again. Also all external shares, via public link or email, will not be accessible.
Internal shares will still be working, so that other users on Nextcloud can continue working.